doc/da65.sgml

```diff
@@ -542,6 +542,18 @@ code. The following attributes are recognized:
   range, where <tt/label/ is the label name given with the <tt/NAME/
   attribute, and <tt/offs/ is the offset within the data.
 
+  <tag><tt>VOPERAND</tt></tag>
+  This optional attribute is followed by a numerical value. It tells the
+  assembler that subroutine calls to this label follow "virtual operands"
```

Comment:
As you can see from my comments below on your sample code that code can be misinterpreted easily. Therefore I suggest that you try to actually describe what VOPERAND does rather just say "like this".